.github/workflows/ci.yml

```diff
@@ -9,14 +9,17 @@ on:
 jobs:
   tests:
     name: Tests
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ ubuntu-latest, windows-latest ]
     steps:
       - uses: actions/checkout@v4
       - uses: dtolnay/rust-toolchain@stable
       - uses: Swatinem/rust-cache@f0deed1e0edfc6a9be95417288c0e1099b1eeec3
       - uses: hrishikesh-kadam/setup-lcov@v1
```

Comment:
nitpick: it's better to use hashes instead of tags for non-official actions
https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions#using-third-party-actions

You can still maintain dependabot functionality (so you never forget to update the actions), you just need to add an inline comment with version tag ([example](https://github.com/software-mansion/scarbs.xyz/pull/409/files))